### PR TITLE
fix imagemagick tests

### LINKED ISSUE #7400: Dev: ImageMagick PHPUnit tests fail
```
convert: unable to read font `helvetica' @ error/annotate.c/RenderFreetype/1396.
convert: unable to read font `helvetica' @ error/annotate.c/RenderFreetype/1396.
convert: unable to read font `helvetica' @ error/annotate.c/RenderFreetype/1396.
```
```
There were 3 errors:

1) CodeIgniter\Images\ImageMagickHandlerTest::testText
CodeIgniter\Images\Exceptions\ImageException: Image processing failed. Please verify that your server supports the chosen protocol and that the path to your image library is correct.

/home/runner/work/CodeIgniter4/CodeIgniter4/system/Images/Handlers/ImageMagickHandler.php:212
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Images/Handlers/ImageMagickHandler.php:408
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Images/Handlers/BaseHandler.php:418
/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Images/ImageMagickHandlerTest.php:300
phpvfscomposer:///home/runner/work/CodeIgniter4/CodeIgniter4/vendor/phpunit/phpunit/phpunit:106

2) CodeIgniter\Images\ImageMagickHandlerTest::testMoreText
CodeIgniter\Images\Exceptions\ImageException: Image processing failed. Please verify that your server supports the chosen protocol and that the path to your image library is correct.

/home/runner/work/CodeIgniter4/CodeIgniter4/system/Images/Handlers/ImageMagickHandler.php:212
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Images/Handlers/ImageMagickHandler.php:408
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Images/Handlers/BaseHandler.php:418
/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Images/ImageMagickHandlerTest.php:308
phpvfscomposer:///home/runner/work/CodeIgniter4/CodeIgniter4/vendor/phpunit/phpunit/phpunit:106

3) CodeIgniter\Images\ImageMagickHandlerTest::testImageCreation
CodeIgniter\Images\Exceptions\ImageException: Image processing failed. Please verify that your server supports the chosen protocol and that the path to your image library is correct.

/home/runner/work/CodeIgniter4/CodeIgniter4/system/Images/Handlers/ImageMagickHandler.php:212
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Images/Handlers/ImageMagickHandler.php:408
/home/runner/work/CodeIgniter4/CodeIgniter4/system/Images/Handlers/BaseHandler.php:418
/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Images/ImageMagickHandlerTest.php:322
phpvfscomposer:///home/runner/work/CodeIgniter4/CodeIgniter4/vendor/phpunit/phpunit/phpunit:106
```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/4570658877/jobs/8068220103

### PR DESCRIPTION
**Pull request by michalsn**
**Description**
Fixes #7400

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Review by samsonasik — CHANGES_REQUESTED** [review]
This workflow path need to be registered to https://github.com/codeigniter4/CodeIgniter4/blob/develop/.github/workflows/test-phpunit.yml so we can see immediate test running

**Comment by michalsn** [comment]
Good point, thanks!

**Review by samsonasik — APPROVED** [review]

**Comment by samsonasik** [comment]
Thank you @michalsn

File: .github/workflows/reusable-phpunit-test.yml [hunk 1]
```diff
@@ -144,8 +144,8 @@ jobs:
         if: ${{ contains(inputs.extra-extensions, 'imagick') }}
         run: |
           sudo apt-get update
-          sudo apt-get install --reinstall libgs9-common fonts-noto-mono libgs9:amd64 libijs-0.35:amd64 fonts-urw-base35 ghostscript poppler-data libjbig2dec0:amd64 gsfonts libopenjp2-7:amd64 fonts-droid-fallback fonts-dejavu-core
-          sudo apt-get install -y imagemagick
+          sudo apt-get install --reinstall libgs9-common fonts-noto-mono libgs9:amd64 libijs-0.35:amd64 fonts-urw-base35 ghostscript poppler-data libjbig2dec0:amd64 libopenjp2-7:amd64 fonts-droid-fallback fonts-dejavu-core
+          sudo apt-get install -y gsfonts libmagickwand-dev imagemagick
```
Comment (kenjis) [inline .github/workflows/reusable-phpunit-test.yml:148]: Awesome! How did you know the fix?

Comment (michalsn) [reply .github/workflows/reusable-phpunit-test.yml:148]: I had it in one of my projects. I don't remember where I got it from... I probably googled it.

**Pull request merged**